deltachat-ios/DC/DcContext.swift

```diff
@@ -388,6 +388,29 @@ public class DcContext {
         }
     }
 
+    public func parseVcard(path: String) -> [DcVcardContact]? {
+        if let data = DcAccounts.shared.blockingCall(method: "parse_vcard", params: [path as AnyObject]) {
```

Comment:
We could put all those strings (`parse_vcard`, `import_vcard`, `send_reaction`) into constants at some point